htdocs/core/class/html.formmail.class.php

```diff
@@ -1473,8 +1476,8 @@ public function getSectionForAIPrompt($format = '')
 					$('#ai_status_message').show();
 
 					//editor on readonly
-        			if (CKEDITOR.instances.message) {
-						CKEDITOR.instances.message.setReadOnly(1);
+        			if (CKEDITOR.instances.".$htmlContent.") {
```

Comment:
As we are using a variable to forge javascript content, you must use 
dol_escape_js($htmlContent)
to invoid js injection through this variable.